cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala

```diff
@@ -325,6 +325,31 @@ class CallSubqueryTest extends DocumentingTest {
       p("""As the size of the CSV file in this example is small, only a single separate transaction is
           #started and committed.""".stripMargin('#'))
       note(p("""`CALL { ... } IN TRANSACTIONS` is only allowed in <<query-transactions, implicit transactions>>."""))
+      section("Deleting a large amount of nodes", "delete-with-call-in-transactions") {
```

Comment:
Should we also change "amount" to "volume" here?

Reply:
Agreed.